### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader:Screen reader  is unnecessarily announcing "Graphics loading" after "External data source for new external table "combo box . 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6075]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5
NVDA version:2019.1
JAWS version : 2019.1810.64 ILM

**Additional Details:**
Tools Name:JAWS
MAS References: MAS1.3.1

**Steps to Reproduce:**

1. Install the app Azure Data Studio.
2. Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files.
3. Navigate till "External data source for new external table "combo box .
4. Observe that screen reader is announcing "loading image" after "External data source for new external table "combo box .

**Actual:**
Screen reader is announcing "Graphics loading" after label element for every combobox. 
for example for "External data source for new external table" combobox, after the label element "External data source for new external table", the "Graphics loading" is announced.

Similar issue is also observed with NVDA.

**Expected:**
Screen Reader should announce the combo box and item list after "External data source for new external table "combo box .It should announce " coriveraTest".

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader only users will get confused and might miss understand the "Graphics loading" as the part of the label element of the combobox.

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6075_[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3306102/6075_.Accessibility.A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
This no longer seems to be an issue in the latest build from master
